cmd/katapult/data_centers.go

```diff
@@ -39,32 +39,44 @@ func dataCentersCmd(client *katapult.Client) *cobra.Command {
 			return nil
 		},
 	}
-	cmd.AddCommand(listCmd)
+}
 
-	getCmd := &cobra.Command{
+func getDataCenterCmd(client dataCentersClient) *cobra.Command {
+	return &cobra.Command{
 		Use:   "get",
 		Args:  cobra.ExactArgs(1),
 		Short: "Get details for a data center",
 		Long:  "Get details for a data center.",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			dcClient := core.NewDataCentersClient(client)
-			dc, resp, err := dcClient.Get(cmd.Context(), core.DataCenterRef{Permalink: args[0]})
+			dc, resp, err := client.Get(cmd.Context(), core.DataCenterRef{Permalink: args[0]})
 			if err != nil {
 				if resp != nil && resp.StatusCode == 404 {
```

Comment:
As of go-katapult 0.1.5 with improved error handling, we no longer need check the response `StatusCode` to know if a 404 not found error was encountered or not. Hence this can be simplified like so:

```suggestion
			dc, _, err := client.Get(cmd.Context(), core.DataCenterRef{Permalink: args[0]})
			if err != nil {
				if errors.Is(err, katapult.ErrNotFound) {
```

Reply:
Also thanks to the coverage report, I noticed this not found condition is never triggered, so I think we should have a test case for when the specific data center does not exist as well.